app/browser/api/ledger.js

```diff
@@ -1852,7 +1881,63 @@ const onCallback = (state, result, delayTime) => {
   return state
 }
 
+const onReferralCodeRead = (err, code) => {
+  if (err) {
+    if (clientOptions.verboseP) {
+      console.error('read error: ' + err.toString())
+    }
+    return
+  }
+
+  if (!code) {
+    return
+  }
+
+  code = code.trim()
+
+  if (code.length > 0) {
```

Comment:
not a big deal but why was this changed back to `> 0` from `=== 6`?

Reply:
because server is currently expecting code 1234 for testing, I change it to this